docs/collectors/PhpFpmCollector.md

```diff
@@ -15,6 +15,15 @@ A sample php-fpm config for this collector to work is
 pm.status_path = /fpm-status
 ```
 
+If the URL of the fpm-status page is http://127.0.0.1:8080/fpm-status, you need to set:
```

Comment:
This is not part of the docstring in `src/collectors/phpfpm/phpfpm.py` so it should be added there since this file is technically autogenerated as noted on lines 1 and 2